inference-engine/src/cldnn_engine/ops/convert_color.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "cldnn_program.h"
+#include "cldnn_common_utils.h"
+
+#include "cldnn/primitives/convert_color.hpp"
+
+namespace CLDNNPlugin {
+
+static void CreateNV12toRGBOp(Program& p, const std::shared_ptr<ngraph::op::v8::NV12toRGB>& op) {
```

Comment:
CreateNV12toRGBOp and CreateNV12toBGROp looks pretty much the same. Could you add common function parameterized with input and output color format?